### ISSUE TITLE
Disable no-spaced-func rule as @typescript-eslint/func-call-spacing replaces it

### ISSUE THREAD

**Issue by pmk1c** [iamturns/eslint-config-airbnb-typescript#246]
Using [`defineEmits` from Vue Setup API](https://v3.vuejs.org/api/sfc-script-setup.html#typescript-only-features), I get an ESLint error for the rule [`no-spaced-func`](https://eslint.org/docs/rules/no-spaced-func).
This rule has been deprecated since ESLint 3.3.0 and is replaced by [`func-call-spacing`](https://eslint.org/docs/rules/func-call-spacing). Therefore it does not have a counterpart in `typescript-eslint`, but it doesn't need one, since there is `@typescript-eslint/func-call-spacing`.

So I think it is safe and better to disable this rule for this package.

Example Code, that gets a false positive:

```
const emit = defineEmits<{
  (e: 'update', value: string): void;
}>();
```
